Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.Events.cs

```diff
@@ -111,7 +111,7 @@ private async void Current_LeavingBackground(object sender, Windows.ApplicationM
         {
             // work around to virtual drawing surface bug.
             await Task.Delay(1000);
-            _drawingSurfaceRenderer.ReDraw(ViewPort);
+            _drawingSurfaceRenderer.ReDraw(ViewPort, _infiniteCanvasScrollViewer.ZoomFactor);
```

Comment:
This feels off. Instead of passing the zoom factor everywhere, create a field inside InfiniteCanvasVirtualDrawingSurface that each of the draw methods there can use. Then automatically set the scale and call redraw when zoom factor is changed

Reply:
ViewPort and ZoomFactor always change at the same time. So we need the current value on every call to redraw (similar to ViewPort). Any suggestions to make this requirement more clear in the code?

I decided for this way as the zoom and view port are linked to each other. Both come from the ScrollViewer and can change at the same time based on the user interaction. That's why it should be considered as a single entity or value pair. It is not related to the screen DPI or canvas size.

To be exact this change fixes blurry drawings if the user zooms into the drawing and is not related to the DPI awareness. (I added this detail to the PR)